libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -32,6 +33,17 @@ const dashboardTable = [
     },
 ];
 
+const assertTableValues = ["Direct Sales", "Inside Sales"];
```

Comment:
separate each input params into one variable, so we can easy use it in some places like `${assertTableValues[0]}`